lua/nvcommunity/folds/origami/init.lua

```diff
@@ -0,0 +1,13 @@
+---@type NvPluginSpec
+local spec = {
+  {
+    "chrisgrieser/nvim-origami",
+    event = "BufReadPost",
```

Comment:
Nope, in their [docs](https://github.com/chrisgrieser/nvim-origami#installation) they mention that the event needs to be `BufReadPost`, otherwise, would prevent saving folds.